lib/std/hash/xxhash.zig

```diff
@@ -0,0 +1,268 @@
+const std = @import("std");
+const mem = std.mem;
+const expectEqual = std.testing.expectEqual;
+
+inline fn rotl(comptime count: comptime_int, value: anytype) @TypeOf(value) {
+    return (value << count) | (value >> (@bitSizeOf(@TypeOf(value)) - count));
+}
```

Comment:
`std.math.rotl`

Reply:
Not sure how I missed that - using `std.math.rotl` now.